sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/Generated/Models/ResourceModelWithAllowedPropertySet.cs

```diff
@@ -50,7 +50,7 @@ internal ResourceModelWithAllowedPropertySet(ResourceIdentifier id, string name,
         /// <summary> The etag field is *not* required. If it is provided in the response body, it must also be provided as a header per the normal etag convention.  Entity tags are used for comparing two or more entities from the same requested resource. HTTP/1.1 uses entity tags in the etag (section 14.19), If-Match (section 14.24), If-None-Match (section 14.26), and If-Range (section 14.27) header fields. </summary>
         public string Etag { get; }
         /// <summary> Gets or sets the identity. </summary>
-        public SystemAssignedServiceIdentity Identity { get; set; }
+        public ResourceModelWithAllowedPropertySetIdentity Identity { get; set; }
```

Comment:
This constant schema change breaks our mechanism of replacing the `ManagedServiceIdentity` type from resourcemanager

Reply:
The reason that this is not replaced:
This type `ResourceModelWithAllowedPropertySetIdentity` has 3 properties: principalId, tenantId and type
The first two are just strings, they will always match. Issues come from the third property `type`.
The property `type` is an enum type, the swagger of this RP is generating from the [common-type/v1](https://github.com/Azure/azure-rest-api-specs/blob/4124b7c2773a714303299f0cfd742b0d26d3bb5d/specification/common-types/resource-management/v1/types.json#L245), it has only one value.
In the meantime, the type we would like it to replace this, is generated from `common-type/v4`, its `type` property has 2 values (None and SystemAssigned)
Therefore this type cannot match now, and it shows like this.
I wonder why this does not match with other identity types in resourcemanager. Maybe because they have more properties? Since others usually has `userAssignedIdentities`